src/System.DirectoryServices/src/System/DirectoryServices/DirectoryVirtualListViewContext.cs

```diff
@@ -24,26 +16,21 @@ internal DirectoryVirtualListViewContext(byte[] context)
         {
             if (context == null)
             {
-                this.context = new byte[0];
+                _context = new byte[0];
```

Comment:
Why did you use `Array.Empty<byte>()` in some places you made this change but not others?  Is there a reason it can't be changed as such here, too?